Exercise-HTML-CSS-1/index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+	<head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        
+		<title>GirlDevelopIt</title>
+		<link href = "css/styles.css" rel = "stylesheet" type ="text/css">
+        
+            <!-- Bootstrap -->
+        <link rel="stylesheet" href="Bootstrap/css/bootstrap.min.css" />
+        <link rel="stylesheet" href="Bootstrap/css/bootstrap.css" />
```

Comment:
As you can see, you are only using 2 bootstrap files here, so you should remove all the unused files that you have in `Bootstrap/css` and also you should check the `js` folder as well.

It also happens that `Bootstrap/css/bootstrap.min.css` and `Bootstrap/css/bootstrap.css` is the same file, the difference is that the first one is compressed (aka. minified, that's where the `min` comes from). The difference is just that the `min` file has lower size and it's used usually in production where you don't need a clean code (just take a lot at that file and you won't understand anything). So just leave this file `bootstrap.min.css` and remove the other one.

In web development there's a usual convention where all the folders are in lower case, so you would rename the folder to `bootstrap`. But also, since you will delete all the unused bootstrap files, you will only end up with `bootstrap.min.css` and `bootstrap.min.js`, so you should store those files in your `css` and `js` folders without creating an extra folder for just bootstrap. And you should do that as well if you ever use an external library as well, because you would only use 1 or 2 files, which will be the minified versions of the library.